content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -284,6 +284,36 @@ Content-Type: application/json
 <followed by regular watch stream starting from resourceVersion="10245">
 ```
 
+## Response compression
+
+{{< feature-state for_k8s_version="v1.16" state="beta" >}}
+
+`APIResponseCompression` is a [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+that allows the API server to compress the responses for **get** and **list** requests, reducing the network
+bandwidth and improving the performance of large-scale clusters. It is enabled by default since Kubernetes 1.16,
+but can be disabled by setting the `--feature-gates` flag on the API server to `APIResponseCompression=false`.
```

Comment:
This paragraph smells like a note about the "feature gate" rather than the feature.
The gates may come and go, but we are more focused on the feature that comes with it.
Suppose the feature gate is GA'ed and/or eventually removed, can we simply remove a few sentences rather than rewriting the paragraph?

Reply:
Done. 

I left _can be disabled by setting the `--feature-gates` flag on the API server to `APIResponseCompression=false`_ because it's one of the asks in #30639:

> Can a client turn it off? Can a client demand a compressed response?